jipdate.py

```diff
@@ -179,6 +179,37 @@ def write_last_jira_comment(f, jira, issue):
         except UnicodeEncodeError:
             vprint("Can't encode character")
 
+def write_one_issue(jira_issues, f, issue):
+    global g_args
+    last_comment = g_args.l
+
+    if issue not in jira_issues:
+        # Add parent as a comment
+        f.write("#[%s] is parent of next issue\n" % issue)
+        f.write("#parent of next issues\n" % issue)
```

Comment:
Ah yes, a last minute changes with a wrong copy/paste to try to make comment more readable.
Will fix it